src/reference/00-Getting-Started/02-sbt-by-example.md

```diff
@@ -10,7 +10,7 @@ out: sbt-by-example.html
 sbt by example
 --------------
 
-This page assumes you've [installed sbt 1][Setup].
+This page assumes you've [installed sbt 1][Setup] and you are running a Bash/UNIX shell on macOS, Linux or WSL ([Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)).
```

Comment:
Why does the system shell matter so much here? Are we trying to preclude Windows console?

Reply:
These instructions would not work on the Windows default console, and some new users might be just confused what this all means and skip the tutorial.

Reply:
And many people still don't know about WSL being available on Windows at all.

Reply:
for example:
![image](https://user-images.githubusercontent.com/2464813/110997482-6000a780-8375-11eb-9157-72ba1bca877b.png)

Reply:
If the user is already familiar with Windows console, I don't want them to switch to using WSL.
I'd much rather document touch equivalent on console together, which is

```
type nul >> build.sbt
```

?